protocol/x/clob/abci.go

```diff
@@ -198,7 +199,14 @@ func PrepareCheckState(
 
 	// 6. Get all potentially liquidatable subaccount IDs and attempt to liquidate them.
 	subaccountIds := liquidatableSubaccountIds.GetSubaccountIds()
-	if err := keeper.LiquidateSubaccountsAgainstOrderbook(ctx, subaccountIds); err != nil {
+	numDeleveragingAttempts, err := keeper.LiquidateSubaccountsAgainstOrderbook(ctx, subaccountIds)
+	if err != nil {
+		panic(err)
+	}
+
+	// 7. Deleverage subaccounts with open positions in final settlement markets.
+	subaccountOpenPositionInfo := make(map[uint32]map[bool]map[subaccounttypes.SubaccountId]struct{})
```

Comment:
This variable is a placeholder for the real information the liquidations daemon will be retrieving